src/app/shared/not-found-page/not-found-page.component.scss

```diff
@@ -0,0 +1,17 @@
+@import '~bootstrap/scss/mixins';/* Importing Bootstrap SCSS file. */
+@import '~bootstrap/scss/functions';/* Importing Bootstrap SCSS file. */
+@import '~bootstrap/scss/variables';/* Importing Bootstrap SCSS file. */
+
+
+:host {
```

Comment:
I think it would be better if you add wrapper class and use text-align: center on this wrapper instead